File: Java/Algorithms/CombSort.java
package Java.Algorithms;

public class CombSort {

    public static void sort(int myarr[]){
        int size = myarr.length;
        int gap = (int) (size/1.3);
        
        for(int i = gap ;i >0 ;i--){
            for (int j =0 ;j<size;j++){
                if ((j+i)>=size) continue;
                if (myarr[j]>myarr[j+i]){
                    int temp = myarr[i+j];
                    myarr[i+j] = myarr[j];
                    myarr[j] = temp;
                }
            }
        }

    }
    public static void main(String[] args) {
        int[] arr = {9,2,7,4,6,5};
        sort(arr);
        for(int val : arr){
            System.out.print(val+" ");
        }
    }
}

File: Python/Algorithms/CombSort.py
def combSort(myarr):
    gap = int(len(myarr)/1.3)

    for i in range(gap,0,-1):
        for j in range(len(myarr)):
            if j+i>=len(arr): continue
            if myarr[j]>myarr[j+i]:
                myarr[j],myarr[j+i] = myarr[j+i],myarr[j]
            


arr = [9,2,7,4,6,5]
combSort(arr)
print(*arr)